content/en/docs/demo/services/ad.md

```diff
@@ -124,4 +124,8 @@ The following auto-instrumented metrics are available for the application:
 
 ## Logs
 
-TBD
+Ad Service uses Log4J and due to the auto instrumentation capabilities of the
+OTel Java agent, the OTel log appender is automatically configured.
```

Comment:
```suggestion
Ad Service uses Log4J, which is automatically configured by the OTel Java agent.
```

Reply:
Avoiding duplication auf "automatic" and also the good old "what is auto instrumentation" question